gatekeeper.py

```diff
@@ -55,9 +65,9 @@ async def on_message(msg):
                     errors += "Missing permissions to send logging message in " + guild.name + "\n"
 
     if len(errors) > 0:
-        await msg.channel.send(f"Completed pre-ban of {user.mention} from {len(client.guilds)} guilds\n\n__Errors__\n" + errors)
+        await msg.channel.send(f"Completed {parsed_content[0].lower()} of {user.mention} from {len(client.guilds)} guilds\n\n__Errors__\n" + errors)
     else:
-        await msg.channel.send(f"Completed pre-ban of {user.mention} from {len(client.guilds)} guilds")
+        await msg.channel.send(f"Completed {parsed_content[0].lower()} of {user.mention} from {len(client.guilds)} guilds")

```

Comment:
I feel like this should put `username (id)` so we can see who it is even after their member data is purged from our clients

Reply:
This makes sense, updating